main.py

```diff
@@ -982,15 +982,15 @@ def run_analysis(self) -> None:
                 exit_app(f"The scan mode {self.scan_mode} is invalid.")
                 return None
 
-            log(f"Copying report templates. Include request headers: {self.report_request_headers}", log_level=LogLevel.DEBUG)
-            os.system("mkdir -p ~/.ZAP_D/reports")
-            os.system("mkdir -p /root/.ZAP_D/reports")
-            if self.report_request_headers is True:
-                os.system("cp -R /zap/reports/traditional-json-headers ~/.ZAP_D/reports/traditional-json")
-                os.system("cp -R /zap/reports/traditional-json-headers /root/.ZAP_D/reports/traditional-json")
-            else:
-                os.system("cp -R /zap/reports/traditional-json ~/.ZAP_D/reports/traditional-json")
-                os.system("cp -R /zap/reports/traditional-json /root/.ZAP_D/reports/traditional-json")
+            # log(f"Copying report templates. Include request headers: {self.report_request_headers}", log_level=LogLevel.DEBUG)
```

Comment:
This could cause problems if the report files don't match the version of zap we're using (or if they've changed directory structure).